boards/xtensa/qemu_xtensa/simcall.h

```diff
@@ -0,0 +1,189 @@
+/*
+ *  simcall.h  -  Simulator call numbers
+ *
+ *  Software that runs on a simulated Xtensa processor using
+ *  the instruction set simulator (ISS) can invoke simulator
+ *  services using the SIMCALL instruction.  The a2 register
+ *  is set prior to executing SIMCALL to a "simcall number",
+ *  indicating which service to invoke.  This file defines the
+ *  simcall numbers defined and/or supported by the Xtensa ISS.
+ *
+ *  IMPORTANT NOTE:  These numbers are highly subject to change!
+ *
+ *  Copyright (c) 2002-2007 by Tensilica Inc.  ALL RIGHTS RESERVED.
+ *  These coded instructions, statements, and computer programs are the
+ *  copyrighted works and confidential proprietary information of Tensilica Inc.
+ *  They may not be modified, copied, reproduced, distributed, or disclosed to
+ *  third parties in any manner, medium, or form, in whole or in part, without
+ *  the prior written consent of Tensilica Inc.
+ */
```

Comment:
That disclaimer sounds distressingly Apache-incompatible...  Are we sure this is kosher?

Reply:
gone